remo/events/static/events/js/events_list.js

```diff
@@ -523,6 +538,16 @@ function send_query(newquery) {
         extra_q += '&campaign__name__iexact=' + initiative;
     }
 
+    if (attendance) {
+        var attendance_range = ATTENDANCE_LEVEL[attendance];
+        if (attendance_range[0]) {
+            extra_q += '&estimated_attendance__gte=' + attendance_range[0];
+        }
+        if (attendance_range[1]) {
```

Comment:
You need both checks in order to get the range. I suggest we keep this as it is.